### PR TITLE
 Added settings layout

### PR DESCRIPTION
**Pull request by vladdezhniuk**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Review by NikolaiYurchenko — COMMENTED** [review]
LG, but few typos and linter issues as extra spaces and missint spaces

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/CoinsList/CoinsListHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <Icon label='icon-more'>
+          <IconMoreHorizontal />
+        </Icon>
+      }
+    >
+      <WalletAddress>
+        <CryptoAddress>{props.walletAddress}</CryptoAddress>
+        <WalletSymbol>
+          <Text size='12px'>ETH</Text>
+          <span />
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/CoinsList/CoinsListHeader.tsx]: what is that?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+    <Wrapper justifyContent='space-between'>
+      <PrefixSuffix onClick={onPrefixClick}>{prefix}</PrefixSuffix>
+      {children}
+      <PrefixSuffix onClick={onSuffixClick}>{suffix}</PrefixSuffix>
+    </Wrapper>
+  )
+}
+
+export default DefaultHeader
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx]: unrelated change?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Currency/template.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      currency: props.currency
+    })
+  }, [])
+
+  useEffect(() => {
+    const { currency, newCurrency } = props
+    if (!isNil(newCurrency) && !equals(currency, newCurrency)) {
+      props.settingsActions.updateCurrency(newCurrency)
+    }
+  })
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Currency/template.tsx:27]: this useEffect should fired on every rerender?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/index.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    })
+  }, [])
+
+  useEffect(() => {
+    const { language, newLanguage } = props
+    if (!isNil(newLanguage) && !equals(language, newLanguage)) {
+      props.settingsActions.updateLanguage(newLanguage)
+      props.preferencesActions.setLanguage(newLanguage, true)
+    }
+  })
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/index.tsx:28]: same

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    <SettingsContainer>
+      <Flex justifyContent='space-between'>
+        {`#${path}` !== window.location.hash ? (
+          <Link to={path} onClick={goBack}>
+            <Icon color='white800' label='IconBack' size='md'>
+              <IconArrowLeft />
+            </Icon>
+          </Link>
+        ) : (
+          <span />
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx:62]: what is that?

Comment (vladdezhniuk) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/index.tsx:62]: I`m sure that linted replaced fragment or something like that
